mechanicalsoup/stateful_browser.py

```diff
@@ -204,17 +204,29 @@ def list_links(self, *args, **kwargs):
         for l in self.links(*args, **kwargs):
             print("    ", l)
 
-    def links(self, url_regex=None, link_text=None, *args, **kwargs):
+    def links(self, *args, **kwargs):
```

Comment:
Unfortunately this breaks backward compatibility for people using `url_regex` as a positional argument.